.gitignore

```diff
@@ -1 +1,6 @@
 .DS_Store
+*.xcworkspacedata
+Tests/AWSTrackingSDKiOSTests/TestConfig.plist
+.swiftpm/xcode/package.xcworkspace/xcuserdata/zeeshansheikh.xcuserdatad/UserInterfaceState.xcuserstate
```

Comment:
Let's avoid merging any changes that are specific to your own username.

Reply:
Gitignore updated